qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -265,13 +263,14 @@ else if(cycle == 4 && base.equals("AC"))
 	 * here we set the base to more the 15 base
 	 * @throws ParserConfigurationException
 	 */
-	public void bothFirstTest() throws ParserConfigurationException {
+	public void bothFirstTest() throws IllegalArgumentException {
```

Comment:
IllegalArgumentException is an unchecked exception and so you don't need it in the throws clause